### PR TITLE
Add T9 style keypad for searching

### PR DESCRIPTION
**Pull request by mrosseel**

### CODE REVIEW

File: python/PiFinder/db/objects_db.py
Comment (jscheidtmann) [inline python/PiFinder/db/objects_db.py:1]: In memory you‘re using `*.lower()`, I would expect a similar construct for accessing DB.

File: python/PiFinder/main.py
Comment (jscheidtmann) [inline python/PiFinder/main.py:1]: Why do you comment out GPS debug messages, here?

Comment (mrosseel) [reply python/PiFinder/main.py:1]: I added them some time ago when debugging gps issues, but they are very noisy and overwhelm all other debugging, so I removed them. Better would be to be able to switch on/off debugging for certain modules, but that's for another PR.

File: python/PiFinder/db/objects_db.py
Comment (mrosseel) [reply python/PiFinder/db/objects_db.py:1]: true, althought the DB version is not used atm. I'll update it to also use lower.

**Comment by brickbots** [comment]
Thank you @mrosseel for this AWESOME new feature 🎉 

@jscheidtmann I appreciate you checking over this PR and adding your questions/thoughts!

**Pull request merged**